maintenancemode/views/defaults.py

```diff
@@ -10,6 +10,4 @@ def temporary_unavailable(request, template_name=MAINTENANCE_503_TEMPLATE):
     table, redirects if found, and displays 404 page if not redirected.
 
     """
-    return HttpResponseTemporaryUnavailable(loader.render_to_string(template_name, {
-        'request_path': request.path,
-    }, RequestContext(request)))
+    return HttpResponseTemporaryUnavailable(loader.render_to_string(template_name, {'request_path': request.path,}))
```

Comment:
We can do something like:
```
if DJANGO_VERSION >= (1, 10, 0)
    return HttpResponseTemporaryUnavailable(loader.render_to_string(template_name, {'request_path': request.path,}))
else
    return HttpResponseTemporaryUnavailable(loader.render_to_string(template_name, {
        'request_path': request.path,
    }, RequestContext(request)))
```
For Django >1.10 both seems to work
**return HttpResponseTemporaryUnavailable(loader.render_to_string(template_name, {'request_path': request.path,}))**
or
**return HttpResponseTemporaryUnavailable(loader.render_to_string(template_name, RequestContext(request)))**
so i dont know if we can use one random.